samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp

```diff
@@ -571,9 +571,6 @@ document.links[newindex].onclick();
 				   && delivery.navigation eq '1' && !delivery.doContinue}" 
       />
 
-  <%-- SUBMIT FOR DUE OR RETRACT DATE --%>
-  <h:commandButton id="submitNoCheck" type="submit" styleClass="hidden active" action="#{delivery.submitFromTimeoutPopup}" value="" />
-
```

Comment:
why remove this?

Reply:
Oh! I have just noticed that this button is called from here too: https://github.com/sakaiproject/sakai/blob/ec8ee1cef6e2d7dffb5908baa492c504d8d65f4e/samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js#L321

I haven't seen that on 20.